x-pack/plugins/enterprise_search/public/applications/shared/constants/health_colors.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const healthColorsMap = {
+  red: 'danger',
+  green: 'success',
+  yellow: 'warning',
+  unavailable: '',
+};
+
+export const healthColorsMapSelectable = {
```

Comment:
Added this variable to support [selectable](x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_creation/search_index_selectable.tsx) option